gym_cap/envs/cap_env.py

```diff
@@ -38,6 +39,62 @@ def __init__(self, map_size=20, mode="random"):
             self.interaction = self._interaction_stoch
         else: self.interaction = self._interaction_determ
 
+    def debug(self, filename, map_obj=[NUM_BLUE, NUM_UAV, NUM_RED, NUM_UAV, NUM_GRAY]):
+    
+        """
+        Debugs the game according to our given inputs
+
+        : filename: contains initialized board
+        : return: void
+        : map_obj     : list
+            The necessary elements to build the map
+            0   : blue UGV
+            1   : blue UAV
+            2   : red UGV
+            3   : red UAV
+            4   : gray units
+
+        """
+
+        # abbreviates filename
+        dir_path = os.path.dirname("C:\\Users\\shamith2\\Desktop\\capture_the_flag\\ctf_public\\tests\\")
+        file_path = os.path.join(dir_path, filename)
+        
+        # imports board from filename as map
+        map = np.loadtxt(file_path, dtype = int, delimiter = " ")
+        self.map_size = map.shape
+        
+        # _env will be the map as in filename
+        # team_home will be the static_map (generated from map in filename)
+        self._env, self.team_home = CreateMap.dest_map(map,\
+                                                       map_obj=[NUM_BLUE, NUM_UAV, NUM_RED, NUM_UAV, NUM_GRAY])
```

Comment:
It would be better if the board condition can override the map_obj numbers

Reply:
I have changed the debug method